enhancements/installer/install_ca_management.md

```diff
@@ -0,0 +1,164 @@
+operators will fail to function, until/unless the admin explicitly configures the CA in the image 
+config resource and then re-imports the imagestream.
+
+While this could theoretically be solved by the imagestream component trusting the additional CA (
+for all hosts?) explicitly, it would make more sense if we used the information the admin is 
+providing us at install time to correctly configure the cluster as if they had done it themselves, 
+rather than introducing additional sources of (potentially conflicting) configuration 
+information.
+
+## Motivation
+
+See summary above, motivation is to make the install experience clearer with less potential for misconfiguration and conflicting settings.
+
+### Goals
+
+- Post install the cluster can not only pull images from an alternate registry, but imagestreams can import from it, builds can pull from it, and the internal registry can perform pullthrough against it
+- Post install the registry CAs provided by the admin are populated into the configuration api that is intended for this purpose
+
+### Non-Goals
+
+- Refactoring the additionalTrustBundle field into many unique fields for different CA usages
+- Addressing the current proxy CA behavior in which the additionalTrustBundle is sometimes populated into a proxy configuration and other times not (depending on whether a proxy host is supplied)
+- Adding generic cluster configuration fields to the installer that are not required to bootstrap the cluster and ensure consistent use of cluster config apis
+
+
+## Proposal
+
+The installer should populate the 
+[image config resource](https://github.com/openshift/api/blob/master/config/v1/types_image.go) in the same way the 
+[product documentation](https://docs.openshift.com/container-platform/4.5/openshift_images/image-configuration.html#images-configuration-cas_image-configuration) 
+recommends.  The installer 
+can use the value of the additionalTrustBundle field to populate this api.  Similar to how the 
+value is used for the proxy config when a proxy host is supplied, the value can be used to 
+populate the image config resource when a mirror registry is in use.
+
+In addition, the downstream components that potentially communicate with the mirror registry should add logic which confirms they can successfully access the mirror registry.  Specifically, for any registry defined in the image content source policy, the components should be able to access the registry using the supplied CAs.  If they cannot, they should mark themselves degraded.
+
+Finally, if possible, the MCO mechanism that injects CAs into the nodes directly should be disabled when the image config api is populated, to ensure we do not have conflicting values being placed on the nodes.  The MCO mechanism should be a pure bootstrapping mechanism.  (That may already be the case today, I am not certain how that mechanism is implemented).
```

Comment:
The installer only have one CA, so MCO during bootstrapping can use that CA (file on disk) + Image (file on disk) to configure the code-path for registries and their trust.
And MCO running in the cluster end up using the Image and then the same CA from k8s API to create the same configuration for the compute machines.
Making the bootstrap and in-cluster results the same.

We will have to keep the legacy behavior of MCO reading the config map directly always unless we migrate all our existing clusters to using Image for registry trust.

Reply:
makes sense.  i'll make a note of why this is not possible.